packages/eslint-config/eslintrc.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
 
     // Avoid promise rewrapping
     // https://exploringjs.com/es2016-es2017/ch_async-functions.html#_returned-promises-are-not-wrapped
-    'no-return-await': 'error',
+    'no-return-await': 'off',
```

Comment:
Can you move it to be next to https://github.com/strongloop/loopback-next/blob/40c70729b1e2b2b350846856f9f63d58933c931f/packages/eslint-config/eslintrc.js#L147 and add a comment to include the link to the docs? Please also remove:

```
// Avoid promise rewrapping
// https://exploringjs.com/es2016-es2017/ch_async-functions.html#_returned-promises-are-not-wrapped
```